### PR TITLE
Update fr.json

### PR DESCRIPTION
**Pull request by MDVAurelien**
Add some French translations

**Changes:**

**Testing:**

### Downstream

|Affects         |Reasoning
|----------------|-------------------------------
|Documentation   |

### CODE REVIEW

File: src/assets/i18n/fr.json [hunk 1]
```diff
@@ -571,10 +571,10 @@
...
   "Release Notes": "",
   "Remote Host Key": "",
-  "Remove this error to try again": "",
-  "Remove {label} item": "",
-  "Remove {value} from recent searches": "",
-  "Renew Certificate Days Before Expiry": "",
+  "Remove this error to try again": "Supprimez cette erreur pour réessayer",
+  "Remove {label} item": "Supprimer l'élément {label}",
+  "Remove {value} from recent searches": "Supprimer {value} des recherches récentes",
+  "Renew Certificate Days Before Expiry": "Supprimer {value} des recherches récentes",
```
Comment (denysbutenko) [inline src/assets/i18n/fr.json:577]: wrong translation, there is no `{value}` placeholder

**Review by denysbutenko — CHANGES_REQUESTED** [review]
please take a look on comment above

**Comment by MDVAurelien** [comment]
Fix my fault

**Review by denysbutenko — APPROVED** [review]
Thanks, looks good!

**Pull request merged**